### PR TITLE
NET6+GraphQLv5

### LINKED ISSUE #125: Update to GraphQL v5
To keep this repo in sync with the latest version of GraphQL, it should be updated to version 5 as well as .NET 6.

### PR DESCRIPTION
**Pull request by sungam3r**
fixes #125 

1. I don't use/know this project (yet).
2. workflows were updated as well as aux files (.editorconfig etc)
3. Both executable projects work (tested by opening /ui/graphiql and sending requests)

### CODE REVIEW

**Comment by sungam3r** [comment]
I'm disabling appveyor.

**Comment by sungam3r** [comment]
I can say that this project has long required attention and good cleaning.

File: src/GraphQL.Relay/GraphQL.Relay.csproj [hunk 1]
```diff
@@ -1,9 +1,12 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
```
Comment (sungam3r) [inline src/GraphQL.Relay/GraphQL.Relay.csproj:4]: @Shane32  change to `<TargetFrameworks>netstandard2.1;netstandard2.0;net6</TargetFrameworks>` ?

File: src/GraphQL.Relay.Todo/Startup.cs [hunk 2]
```diff
@@ -1,41 +1,29 @@
...
+            services.AddGraphQL(b => b
+                .AddServer(true)
+                .AddHttpMiddleware<TodoSchema>()
+                .AddSchema<TodoSchema>()
+                .AddSystemTextJson()
+                .AddErrorInfoProvider(options => options.ExposeExceptionStackTrace = true)
+                .AddSystemTextJson());
         }
 
         public async void Configure(IApplicationBuilder app, IWebHostEnvironment env)
```
Comment (sungam3r) [inline src/GraphQL.Relay.Todo/Startup.cs:22]: 😮 async void !

File: .github/labeler.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+test:
+  - src/GraphQL.Relay.Test/**/*
+  - src/GraphQL.Relay.ApiTests/**/*.cs
```
Comment (sungam3r) [inline .github/labeler.yml:3]: will be added later

**Comment by sungam3r** [comment]
CodeCov works!

File: .github/labeler.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  - .github/workflows/**/*
+  - .github/dependabot.yml
+  - .github/labeler.yml
+  - .github/codecov.yml
+
+code style:
+  - .editorconfig
+
+documentation:
+  - README.md
```
Comment (Shane32) [inline .github/labeler.yml:15]:
```suggestion
```
Please remove labeler

File: .github/workflows/test-code.yml [hunk 5]
```diff
@@ -27,34 +27,48 @@ jobs:
...
-      - name: Setup .NET Core 5.0 SDK
-        uses: actions/setup-dotnet@v1
+        uses: actions/checkout@v3
+      - name: Setup .NET Core SDKs
+        uses: actions/setup-dotnet@v2
         with:
-          dotnet-version: '5.0.x'
+          dotnet-version: |
+            5.0.x
+            6.0.102
```
Comment (Shane32) [inline .github/workflows/test-code.yml:43]: Personally I would only test with 6.0 for this project.  If anything else, 3.1 which is also a LTS release, but not 5.0.

File: GraphQL.Relay.sln [hunk 6]
```diff
@@ -11,18 +11,29 @@ EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.Relay.Todo", "src\GraphQL.Relay.Todo\GraphQL.Relay.Todo.csproj", "{15AFC7EC-11E6-469F-87DF-D1E396463BE9}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".github", ".github", "{2F92B771-95CF-4FC3-AE9A-52E9273349F1}"
+	ProjectSection(SolutionItems) = preProject
+		.github\codecov.yml = .github\codecov.yml
+		.github\dependabot.yml = .github\dependabot.yml
+		.github\labeler.yml = .github\labeler.yml
```
Comment (Shane32) [inline GraphQL.Relay.sln:17]:
```suggestion
```

File: src/GraphQL.Relay/GraphQL.Relay.csproj (hunk 1 above)
Comment (Shane32) [reply src/GraphQL.Relay/GraphQL.Relay.csproj:4]: I would not unless we have an ifdef that requires it.  Nuget should automatically pull dependencies appropriately for the correct TFM regardless of what this package's target framework is.

**Review by Shane32 — CHANGES_REQUESTED** [review]

**Comment by sungam3r** [comment]
All done.

**Review by Shane32 — APPROVED** [review]

**Pull request merged**